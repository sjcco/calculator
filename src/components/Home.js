import React from 'react';

const Home = () => (
  <>
    <h1>Welcome to our page!</h1>
    <p>
      Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aliquid itaque eum, non quis sit et
      dicta vitae quae asperiores expedita, error quos suscipit.
    </p>
    <p>
      Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aliquid itaque eum, non quis sit et
      dicta vitae quae asperiores expedita, error quos suscipit.
    </p>
  </>
);

export default Home;
