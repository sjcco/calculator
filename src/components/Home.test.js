import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

test('render page correctly', () => {
  const home = renderer.create(<Home />).toJSON();
  expect(home).toMatchSnapshot();
});
