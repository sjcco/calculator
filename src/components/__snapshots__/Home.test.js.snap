// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render page correctly 1`] = `
Array [
  <h1>
    Welcome to our page!
  </h1>,
  <p>
    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aliquid itaque eum, non quis sit et dicta vitae quae asperiores expedita, error quos suscipit.
  </p>,
  <p>
    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aliquid itaque eum, non quis sit et dicta vitae quae asperiores expedita, error quos suscipit.
  </p>,
]
`;
